### PR TITLE
issuefix #182: sorts saved forms by title in ascending order

### PR DESCRIPTION
**Pull request by rockydcoder**

### CODE REVIEW

File: public/javascripts/modals.js [hunk 1]
```diff
@@ -37,6 +37,10 @@ var modalsNS = odkmaker.namespace.load('odkmaker.modals');
             {
                 $dialog.find('.modalLoadingOverlay').stop().fadeOut();
 
+                odkmaker.auth.currentUser.forms.sort(function(a, b) {
```
Comment (rockydcoder) [inline public/javascripts/modals.js]: Would it make sense to put the length check before doing anything with the list?

**Comment by clint-tseng** [comment]
I will look at this and get back to you tomorrow!

**Comment by rockydcoder** [comment]
Gentle remainder to review this, @clint-tseng

File: public/javascripts/modals.js (hunk 1 above)
Comment (clint-tseng) [reply public/javascripts/modals.js]: I think this is fine as-is; an array is guaranteed and sorting and empty array is a free no-op.

**Comment by clint-tseng** [comment]
Apologies on the late review. Been heads down on another ODK project.
My only complaint, and this is arguably not a real problem but I'm going to flag it anyway: I am somewhat uncomfortable with mutating the forms array in-place. I'd feel better if a copy was used. `.slice(0).sort()` is fine.

File: public/javascripts/modals.js [hunk 2]
```diff
@@ -37,12 +37,16 @@ var modalsNS = odkmaker.namespace.load('odkmaker.modals');
             {
                 $dialog.find('.modalLoadingOverlay').stop().fadeOut();
 
-                _.each(odkmaker.auth.currentUser.forms, function(formObj)
+                var sortedForms = odkmaker.auth.currentUser.forms.slice(0).sort(function(a, b) {
```
Comment (rockydcoder) [inline public/javascripts/modals.js]: @clint-tseng , did you mean this?

**Comment by clint-tseng** [comment]
Yup! Thanks.

**Pull request merged**